test/powershell/Modules/Microsoft.PowerShell.Utility/Json.Tests.ps1

```diff
@@ -345,6 +345,253 @@ Describe "Json Tests" -Tags "Feature" {
+			$result."about" | Should -Beoftype [String]
+			$result."address"| Should -Be "931 Kings Place, Hartsville/Hartley, Federated States Of Micronesia, 9344"
+			$result."address" | Should -Beoftype [String]
+			$result."age" | Should -Beoftype [Int64]
+			$result."balance"| Should -Be ",039.72"
+			$result."balance" | Should -Beoftype [String]
+			$result."company"| Should -Be "INSECTUS"
+			$result."company" | Should -Beoftype [String]
+			$result."email"| Should -Be "rhodesroberson@insectus.com"
+			$result."email" | Should -Beoftype [String]
+			$result."eyeColor"| Should -Be "green"
+			$result."eyeColor" | Should -Beoftype [String]
+			$result."favoriteFruit"| Should -Be "banana"
+			$result."favoriteFruit" | Should -Beoftype [String]
+			$result."gender"| Should -Be "male"
+			$result."gender" | Should -Beoftype [String]
+			$result."greeting"| Should -Be "Hello, Rhodes Roberson! You have 9 unread messages."
+			$result."greeting" | Should -Beoftype [String]
+			$result."guid"| Should -Be "429b96a7-24e3-47de-a93b-f44a346c5ac9"
+			$result."guid" | Should -Beoftype [String]
+			$result."index" | Should -Beoftype [Int64]
+			$result."isActive"| Should -Be False
+			$result."isActive" | Should -Beoftype [Boolean]
+			$result."latitude"| Should -Be 51.890798
+			$result."latitude" | Should -Beoftype [Double]
+			$result."longitude"| Should -Be -47.522764
+			$result."longitude" | Should -Beoftype [Double]
+			$result."name"| Should -Be "Rhodes Roberson"
+			$result."name" | Should -Beoftype [String]
+			$result."phone"| Should -Be "+1 (883) 561-3999"
+			$result."phone" | Should -Beoftype [String]
+			$result."picture"| Should -Be "http://placehold.it/32x32"
+			$result."picture" | Should -Beoftype [String]
+			$result."registered"| Should -Be "2019-12-17T06:14:06 +06:00"
+			$result."registered" | Should -Beoftype [String]
+			$result."_id"| Should -Be "60dd3ea9253016932039a0a2"
+			$result."_id" | Should -Beoftype [String]
+			
+			$result.Tags | Should -Beoftype [string]
+			
+			$result.Tags.count | Should -Be 7 
+			$result.Tags[0] | Should -Be "laboris"
+			$result.Tags | Should -Be @("laboris", "voluptate", "amet", "ad", "velit", "ipsum", "do")
+			
+			$result.Friends | Should -Beoftype [pscustomobject]
+			$result.Friends[0].id | Should -Be 0
+			$result.Friends[0].name | Should -Be "Renee Holden"
+			$result.Friends[1].id | Should -Be 1
+			$result.Friends[1].name | Should -Be "Bennett Dixon"
+			$result.Friends[2].id | Should -Be 2
+			$result.Friends[2].name | Should -Be "Emilia Holder"
```

Comment:
For information only. If case is important we use `Should -BeExactly`.

Reply:
Is there any good reason for me not to change all of the string checks to BeExactly?

Reply:
You can do that.
This would be especially important if we used different serialization policies for property case (like Pascal or Camel) in tests.